dashboard/lib/account_purger.rb

```diff
@@ -57,9 +57,11 @@ def purge_data_for_email(_email)
   private def really_purge_data_for_account(user)
     ActiveRecord::Base.transaction do
       PEGASUS_DB.transaction do
-        DeleteAccountsHelper.
-          new(bypass_safety_constraints: @bypass_safety_constraints, log: @log).
-          purge_user user
+        DASHBOARD_DB.transaction do
```

Comment:
Oof, this seems scary.  I hope this doesn't cause a deadlock of some sort.  Maybe have someone from Foundation take a quick look and make sure this is ok?

Reply:
This was already wrapped in an active record and pegasus transaction and was tested in unit tests, to keep existing functionality now that we're querying dashboard with sequel, this would need to be added. But yes, let's have foundations look at it. @code-dot-org/foundations

Reply:
Here's where this is tested https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/lib/account_purger_test.rb#L84

Reply:
cc @sureshc